src/care/ProfilePlugin.h

```diff
@@ -0,0 +1,46 @@
+#ifndef _CARE_ProfilePlugin_H_
+#define _CARE_ProfilePlugin_H_
+
+#include "RAJA/util/PluginStrategy.hpp"
+#include "chai/ExecutionSpaces.hpp"
+#include "care/config.h"
+#include "care/CHAICallback.h"
+#include "care/RAJAPlugin.h"
+
+using namespace care;
```

Comment:
"using namespace" is a really bad idea in a header file. It can create function name and class name conflicts for anything that uses CARE.